### PR TITLE
chore: add prettier to CI checks

### PR DESCRIPTION
**Pull request by himeshladva**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
```
Comment (lornest) [inline .prettierrc:4]: yay

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "singleQuote": true
```
Comment (lornest) [inline .prettierrc:5]: yay

File: plugins/terraform-backend/src/schema/openapi.generated.ts [hunk 3]
```diff
@@ -3,136 +3,135 @@
 // ******************************************************************
 // * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. *
 // ******************************************************************
-import { createValidatedOpenApiRouter } from '@backstage/backend-openapi-utils';
+import {createValidatedOpenApiRouter} from '@backstage/backend-openapi-utils';
```
Comment (lornest) [inline plugins/terraform-backend/src/schema/openapi.generated.ts:6]: boo

**Review by lornest — DISMISSED** [review]
LGTM. Prefer the spaces inside  { } personally but happy to concede to the prettier default

**Review by BillyMichael — APPROVED** [review]

File: plugins/terraform-backend/src/schema/openapi.generated.ts (hunk 3 above)
Comment (himeshladva) [reply plugins/terraform-backend/src/schema/openapi.generated.ts:6]: this is ignored by prettier as it's a generated file

**Comment by himeshladva** [comment]
> LGTM. Prefer the spaces inside { } personally but happy to concede to the prettier default

It's a generated file so I added it to `.prettierignore` and regenerated it. Spaces are inside `{}` everywhere else!

**Review by lornest — APPROVED** [review]

**Comment by lornest** [comment]
Approved!

**Pull request merged**